scripts/e2e/init-template-e2e.js

```diff
@@ -59,6 +61,7 @@ async function main() {
 
   Options:
     --projectName             The name of the new React Native project.
+    --currentBranch           The current branch to checkout.
```

Comment:
nit: this is a bit unclear. We should clarify that you're referring to the branch of the template repo